#!/usr/bin/env python
# Created on: Sept 2, 2012
#
#       Author: Pablo Insigo Blasco - Robotics and Computer Architecture (ATC) - University of Seville - 2012
#       License: GPLv3
#http://www.ros.org/wiki/dynamic_reconfigure/Tutorials/HowToWriteYourFirstCfgFile

PACKAGE = "stage_rep"
import roslib;roslib.load_manifest(PACKAGE)

#http://www.ros.org/wiki/dynamic_reconfigure/Tutorials/HowToWriteYourFirstCfgFile
from dynamic_reconfigure.parameter_generator import *
gen = ParameterGenerator()
gen.add("map_filepath", str_t, 0, "The path of the file of the simulator map.", "")
exit(gen.generate(PACKAGE, "stage", "StageSimulator"))
